File: SpringLogicServer/src/main/java/dk/groupfive/SpringLogicServer/model/tasks/PlaceTask.java
package dk.groupfive.SpringLogicServer.model.tasks;


import dk.groupfive.SpringLogicServer.model.objects.Place;

import java.io.Serializable;

public class PlaceTask implements Serializable {
    private String taskName;
    private Place place;
    private long placeID;

    public PlaceTask(String taskName, Place place) {
        this.taskName = taskName;
        this.place = place;
    }

    public PlaceTask(String taskName, long id) {
        this.taskName = taskName;
        placeID = id;
    }

    public String getTaskName() {
        return taskName;
    }

    public Place getPlace() {
        return place;
    }

    public long getPlaceID() {
        return placeID;
    }
}

File: Client/Client/Models/PlaceTask.cs
﻿using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Client.Models
{
	[Serializable]
	public class PlaceTask
	{
		public string taskName { get; set; }
		public Place place { get; set; }
		public long placeID { get; set; }

		public PlaceTask(string taskName, Place place, long placeID)
		{
			this.taskName = taskName;
			this.place = place;
			this.placeID = placeID;
		}
	}
}